orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/model/Stage.java

```diff
@@ -803,6 +803,17 @@ public void setAllowSiblingStagesToContinueOnFailure(boolean propagateFailuresTo
     context.put("allowSiblingStagesToContinueOnFailure", propagateFailuresToParent);
   }
 
+  @JsonIgnore
+  public void setContinuePipelineOnFailure(boolean continuePipeline) {
+    context.put("continuePipeline", continuePipeline);
+  }
+
+  @JsonIgnore
+  public boolean getContinuePipelineOnFailure() {
+    StageContext context = (StageContext) getContext();
```

Comment:
slightly off topic but what's with the cast here? Can we add a `getStageContext` method that returns a `StageContext`?

Reply:
yeah, that would be better, i will add to a list of things for me to fix, thanks!